client/blocks/post-share/index.jsx

```diff
@@ -438,7 +438,7 @@ class PostShare extends Component {
 		if ( failure ) {
 			return (
 				<Notice status="is-error" onDismissClick={ this.dismiss }>
-					{ translate( 'Something went wrong. Please don\'t be mad.' ) }
+					{ translate( 'Something went wrong. Please try again later.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Something went wrong. Please try again.' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/show-error-message-on-publicize-error)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).